.github/workflows/unit-tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false  # ensures the entire test matrix is run, even if one permutation fails
       matrix:
         os: [ windows-latest, ubuntu-latest ]
-        version: [ net462, net6.0, net7.0, net8.0 ]
+        version: [ net462, net6.0, net8.0 ]
```

Comment:
does it make sense to still have version 6? Having 6 and 8 can be confusing, why not start at 8?

Reply:
.NET 6 and 8 are a LTS releases, with .NET 6 support ending in November 2024.

https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core

At one point it was common to use [.NET Standard](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0) as a target, but the explicit runtime versions are preferred now.

Reply:
Ah I see, thanks for explaining :)